src/utils/dateUtils.ts

```diff
@@ -0,0 +1,5 @@
+import { endOfToday, formatISO } from 'date-fns';
+
+export const todayIsoString = () => {
+    return formatISO(endOfToday());
+};
```

Comment:
Her kan den returneres direkte
```suggestion
export const todayIsoString = () => 
    formatISO(endOfToday());
```